code/modules/mob/living/carbon/brain/brain_item.dm

```diff
@@ -68,11 +60,17 @@
 
 /obj/item/organ/internal/brain/examine(mob/user)
 	..()
-	if(brainmob && brainmob.client)
-		user << "You can feel the small spark of life still left in this one."
-	else
-		user << "This one seems particularly lifeless. Perhaps it will regain some of its luster later."
 
+	if(brainmob)
+		if(brainmob.client)
+			if(brainmob.health <= config.health_threshold_dead)
+				user << "It's lifeless and severely damaged."
+			else
+				user << "You can feel the small spark of life still left in this one."
+		else
+			user << "This one seems particularly lifeless. Perhaps it will regain some of its luster later."
+	else
+		user << "This one is completely devoid of life."

```

Comment:
I added a message from brain organ without brain mob, and for the brain organ whose brain mob has been beaten to a pulp.